src/progly/query.cc

```diff
@@ -404,7 +418,96 @@ void worker_exec_runstats_op(librados::IoCtx *ioctx, stats_op op)
   ioctx->close();
 }
 
+void worker_init_lock_obj_op(librados::IoCtx *ioctx, inbl_lockobj_info op)
+{
+    std::string oid = op.table_group;
```

Comment:
append .lock to tablegroupname. so the lock obj name is always tablegroupname.lock